pkg/k8shandler/collection.go

```diff
@@ -5,36 +5,42 @@ import (
 	"io/ioutil"
 	"reflect"
 
+	"strings"
+	"time"
+
 	"github.com/openshift/cluster-logging-operator/pkg/utils"
 	"github.com/sirupsen/logrus"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
fyi this is unnecessary, the defaulted name it gets is the last part of the package which is already "v1"

Reply:
ack.  this is copy/past from some other file

Reply:
interestingly, this is being reverted for me.  I wonder if there is some intelligence to 'go fmt' that is adding it back in